src/tpm2_wrap.c

```diff
@@ -1170,12 +1188,21 @@ int wolfTPM2_SensitiveToPrivate(TPM2B_SENSITIVE* sens, TPM2B_PRIVATE* priv,
         Aes enc;
         TPM2B_IV ivField;
         TPM2B_SYM_KEY symKey;
+        TPM2_CTX* ctx = TPM2_GetActiveCtx();
 
         /* Generate IV */
         ivField.size = digestSz;
-        rc = TPM2_GetNonce(ivField.buffer, ivField.size);
-        if (rc != 0)
+        rc = TPM2_AcquireLock(ctx);
```

Comment:
I do not understand why is active context needed here? 
Is it because of TPM2_GetNonce? 
If yes, shouldn't then the mutex be acquired inside TPM2_GetNonce?